config/job.js

```diff
@@ -72,16 +72,22 @@ const SCHEMA_REQUIRES = Joi.alternatives().try(
     Joi.array().items(SCHEMA_REQUIRES_VALUE),
     SCHEMA_REQUIRES_VALUE
 );
+const SCHEMA_SOURCEPATH = Joi.string().max(100).optional();
+const SCHEMA_SOURCEPATHS = Joi.alternatives().try(
+    Joi.array().items(SCHEMA_SOURCEPATH),
+    SCHEMA_SOURCEPATH
```

Comment:
Are we really allowing sourcePaths to be either a string or an array? There's no test for this use case.

Reply:
Added a test